msteams-platform/bots/how-to/Teams conversational AI/conversation-ai-quick-start.md

```diff
@@ -168,22 +230,24 @@ The bot uses the GPT turbo 3.5 model to chat with Teams users and respond in a p
     The tunnel you created is listed under **Dev Tunnels > (name of the tunnel)**.
 
 1. Go to **Solution Explorer** and select your project.
+
 1. Right-click the menu and select **Teams Toolkit** > **Prepare Teams App Dependencies**.
 
-   :::image type="content" source="../../../assets/images/bots/dotnet-ai-library-prepare-teams-app.png" alt-text="Screenshot shows an example of the Prepare Teams app Dependencies option under Teams Toolkit section in Visual Studio.":::
+   :::image type="content" source="../../../assets/images/bots/dotnet-ai-library-prepare-teams.png" alt-text="Screenshot shows an example of the Prepare Teams app Dependencies option under Teams Toolkit section in Visual Studio.":::
 
-   If prompted, sign in to your Microsoft 365 account. You'll receive a message that Teams app is successfully prepared.
+   If prompted, sign in to your Microsoft 365 account. You receive a message that Teams app is successfully prepared.
 
 1. Select **OK**.
 
 1. Select **F5** or select **Debug** > **Start**.
+
 1. Select **Add**. The app is added to Teams and a chat window opens.
 
-   :::image type="content" source="../../../assets/images/bots/dotnet-ai-library-add-app.png" alt-text="Screenshot shows the add option to add the app to Microsoft Teams.":::
+   :::image type="content" source="../../../assets/images/bots/lightbot-add.png" alt-text="Screenshot shows adding the lightbot app.":::
 
 1. In the message compose area, send a message to invoke the bot.
 
-   :::image type="content" source="../../../assets/images/bots/dotnet-ai-library-invoke-chef-bot.png" alt-text="Screenshot shows an example of a chat window and a message from the chef bot as a reply to users message.":::
+   :::image type="content" source="../../../assets/images/bots/lightbot-output.png" alt-text="Screenshot shows an example of the lightbot output.":::
```

Comment:
Same for this image, doesn't showup.

Reply:
Could you please clear the cache and give it a try? The image appears to be rendering correctly on my end.
<img width="437" alt="image" src="https://github.com/MicrosoftDocs/msteams-docs/assets/129953098/e5999099-760f-4ae4-b594-a9f8385d6a0f">